config/config.go

```diff
@@ -52,6 +53,9 @@ const (
 	// DefaultServerTLSKey is the default path to use when looking for the Buildkit TLS key
 	DefaultServerTLSKey = "./certs/buildkit_key.pem"
 
+	// DefaultSatelliteCertsDir is the default path to use for saving TLS certs and keys for self-hosted satellites
+	DefaultSatelliteCertsDir = "./certs"
```

Comment:
do you mind checking if this is set in the right full path when specifying a different installation_dir (basically that the concatenation occurs after installation_dir and the cfg are set) ?